src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  caregories: [],
+};
+
+export const catagoriySlice = createSlice({
+  name: 'catagory',
+  initialState,
+  reducers: {
+    status: (state) => {
+      state.status = 'under construction';
+    },
```

Comment:
- [ ] @BranBayou great work implementing the changes requested, however in this file, the reducer is wrong , it should give an alternative to when the state is not status, also would recommend change of the reducer , take a look at the example below:
 ```suggestion
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },

```